.generation/post-process/typescript.py

```diff
@@ -0,0 +1,104 @@
+#!/bin/python3
```

Comment:
Do we need to override the user agent here too?

Reply:
[f4fdd3e](https://github.com/geo-engine/openapi-client/pull/1/commits/f4fdd3e377d9ca286d52e0ea72e931077486cf53)